src/openforms/prefill/exceptions.py

```diff
@@ -0,0 +1,10 @@
+class PrefillSkippedException(Exception):
+    """
+    Custom exception for prefill plugins
+
+    This is used in cases where the plugin, for some reason, doesn't manage to complete
+    the operation. This is happening before even making an external call, which can cause
+    another Exception.
```

Comment:
```suggestion
    Raise this exception in a plugin if the preconditions or business logic lead to the
    prefill not being necessary, e.g. a certain authentication attribute is not available or
    configuration of the plugin results in no operation needing to be performed. If
    the implementation decides to not make any external calls to perform the prefill,
    then you should raise this exception.
    
    Do not raise this exception in case of prefill failure - those errors are handled 
    separately.
```